Source/JavaScriptCore/wasm/WasmSectionParser.cpp

```diff
@@ -559,30 +559,37 @@ auto SectionParser::parseElement() -> PartialResult
             Element element(Element::Kind::Passive, TableElementType::Funcref);
             WASM_PARSER_FAIL_IF(!element.functionIndices.tryReserveCapacity(indexCount), "can't allocate memory for ", indexCount, " Element indices");
 
-            WASM_FAIL_IF_HELPER_FAILS(parseElementSegmentVectorOfExpressions(element.functionIndices, indexCount, elementNum));
+            WASM_FAIL_IF_HELPER_FAILS(parseElementSegmentVectorOfExpressions(TableElementType::Funcref, element.functionIndices, indexCount, elementNum));
             m_info->elements.uncheckedAppend(WTFMove(element));
             break;
         }
         case 0x06: {
```

Comment:
It's a little sad both here and the spec don't have named values for these cases.

Reply:
Yeah, this is unfortunate...